integration/compareDissecCentralized.spec.js

```diff
@@ -0,0 +1,101 @@
+const { execSync } = require('child_process')
+const {
+  default: CozyClient,
+  createClientInteractive,
+  Q
+} = require('cozy-client')
+
+const { populateCentralized } = require('../scripts/populateCentralized')
+const { dissecLearning } = require('./helpers/dissecLearning')
+const { localLearning } = require('./helpers/localLearning')
```

Comment:
I don't think dissecLearning and localLearning can be seen as `helpers`. A `learning` directory sounds more relevant.